### PR TITLE
Wire up secondary nics

### PR DESCRIPTION
**Pull request by qinqon**
This way they are wired up so they can communicate between nodes

### CODE REVIEW

**Comment by slintes** [comment]
I'm not very familiar with network, but AFAIU this makes sense to me.
Waiting for the updates image shas.
/lgtm

**Comment by qinqon** [comment]
@slintes I may also need to run dnsmasq per addintional nic o assign ip addresses and all, but not sure yet I have to test with generated containers. If I need it new PR will be added, but this PR is good as it is.

btw updating containers is soooo painful.

**Comment by slintes** [comment]
> btw updating containers is soooo painful.

Haha, it was painful before we had the prow jobs for publishing them! :wink: 
But yes, maybe we can automate pushing images + updating shasums somehow.

**Comment by qinqon** [comment]
@slintes now that I think of... I have change base image dnsmasq.sh is there so do I have to push base too and change references ?

**Comment by slintes** [comment]
ooohhh... good catch, I need more coffee :coffee:
Yes, you need to first rebuild base, then centos, and then the cluster images
And for base and centos we don't have prow jobs yet, will create them right now
And you also modified gocli...

**Comment by qinqon** [comment]
Huff that's crazy :-)

**Comment by slintes** [comment]
@qinqon Here we go, new jobs are created.
For the base image you can use this sha from my  manual test run of that job: `sha256:59f2a244d7f2b966f9f1b8a9b4d9286b779027f774201514c4f27afd0e7bc585`
With that you need to rebuild centos7 with `/test push-centos7`.
After that you can push the cluster images again.
And gocli.
Sorry that this is indeed a suboptimal process yet.

**Comment by qinqon** [comment]
Ok last step... few

**Comment by qinqon** [comment]
@slintes all shas done, I am older now :-)

File: cluster-provision/README.md [hunk 1]
```diff
@@ -16,16 +16,16 @@
 
 * `kubevirtci/cli`: `sha256:1dd015dea4f12e6dcb8e31be3eeb677fed96f290ef4a4892a33c43d666053536`
 * `kubevirtci/gocli`: `sha256:1e21a7b0fc959ae2a47d1d6462ceb74f9e9181d52f9bc618e61bcab80bedaa9e`
-* `kubevirtci/base`: `sha256:034de1a154409d87498050ccc281d398ce1a0fed32efdbd66d2041a99a46b322`
-* `kubevirtci/centos:1905_01`: `sha256:f2b204a3fabf71494c3cf7a145dae88e63f068a84ca10149b0aec523045dc2c1`
-* `kubevirtci/os-3.11.0-multus`: `sha256:f8de6bda57209553e1a9e428dca1432270b7db0e5206376898b93aea64afd053`
+* `kubevirtci/base`: `sha256:59f2a244d7f2b966f9f1b8a9b4d9286b779027f774201514c4f27afd0e7bc585`
+* `kubevirtci/centos:1905_01`: `sha256:a02a1253a4bb734ee7fb244b42afcc7fdc66f5e26f440eb718876f1913e1717a`
+* `kubevirtci/os-3.11.0-multus`: `sha256:9b6f0f8535a82515960cc48bc68a49857206dfe57426d3ad383411d7752b1c8c`
 * `kubevirtci/os-3.11.0:`: `sha256:527daa78a9e460053925a862324f67390d45ffacaf26fe3fd5a6dc20a7892534`
```
Comment (slintes) [inline cluster-provision/README.md]: seems you forgot os-3.11.0, should be sha256:fb5d586c62887f2f5e1ddeeae05da94c28baf4bcc27201e543afee3984b430b0

File: cluster-provision/README.md [hunk 2]
```diff
@@ -16,16 +16,16 @@
 
 * `kubevirtci/cli`: `sha256:1dd015dea4f12e6dcb8e31be3eeb677fed96f290ef4a4892a33c43d666053536`
 * `kubevirtci/gocli`: `sha256:1e21a7b0fc959ae2a47d1d6462ceb74f9e9181d52f9bc618e61bcab80bedaa9e`
-* `kubevirtci/base`: `sha256:034de1a154409d87498050ccc281d398ce1a0fed32efdbd66d2041a99a46b322`
-* `kubevirtci/centos:1905_01`: `sha256:f2b204a3fabf71494c3cf7a145dae88e63f068a84ca10149b0aec523045dc2c1`
-* `kubevirtci/os-3.11.0-multus`: `sha256:f8de6bda57209553e1a9e428dca1432270b7db0e5206376898b93aea64afd053`
+* `kubevirtci/base`: `sha256:59f2a244d7f2b966f9f1b8a9b4d9286b779027f774201514c4f27afd0e7bc585`
+* `kubevirtci/centos:1905_01`: `sha256:a02a1253a4bb734ee7fb244b42afcc7fdc66f5e26f440eb718876f1913e1717a`
+* `kubevirtci/os-3.11.0-multus`: `sha256:9b6f0f8535a82515960cc48bc68a49857206dfe57426d3ad383411d7752b1c8c`
```
Comment (slintes) [inline cluster-provision/README.md]: you picked up the wrong sha from the logs, should be sha256:d4a4b62e853b5f4b4c5b37fbdb369327a060b9caec48a77bf9cf2618bc6330fe

File: cluster-provision/README.md [hunk 3]
```diff
@@ -16,16 +16,16 @@
...
 * `kubevirtci/os-3.11.0:`: `sha256:527daa78a9e460053925a862324f67390d45ffacaf26fe3fd5a6dc20a7892534`
-* `kubevirtci/os-3.11.0-crio:`: `sha256:5eb4a00c89a9dcadccd3360a43973ac38897511260ce192598d6d8731c0c3e7a`
+* `kubevirtci/os-3.11.0-crio:`: `sha256:fbeab9dfbabb9ff892fc54ad043eb8bbf8ba7b9ac1096b99b417e240b3f4a708`
```
Comment (slintes) [inline cluster-provision/README.md]: sha256:e8511fb8aa77e30527f5ee3cb2990e886680eccf7770787504f530e7fdf5d402

File: cluster-provision/README.md [hunk 4]
```diff
@@ -16,16 +16,16 @@
...
 * `kubevirtci/k8s-1.10.11:`: `sha256:f563a8ab4719e53c2372c4f41dfe55256677ec7afc442dfaebd494926005e3e5`
-* `kubevirtci/k8s-1.11.0:`: `sha256:2104f70ac71cee8985c0cb69eacab41bd706e6e5f6afbcc91b31a34c716871ac`
-* `kubevirtci/k8s-1.13.3:`: `sha256:1b6211919c9b1b2bb79d7c92a233721ae504f1ff17a111a897e3c3ceb64f3cd0`
-* `kubevirtci/k8s-multus-1.13.3:`: `sha256:4ceee57c99649df85664dd0434b1f98c69c2d75012110c4e7c8f6b5c41ce4b25`
-* `kubevirtci/k8s-genie-1.11.1:`: `sha256:f61813c7c41cdd51f235b28fb574a39dc71e421c388dc85c4019912d4a4421bb`
+* `kubevirtci/k8s-1.11.0:`: `sha256:3554a97f48fda01a1e83ff8663eb8f42bfff46745ad91d808650c309651669c9`
```
Comment (slintes) [inline cluster-provision/README.md]: sha256:9528a2909d1119ab7ff8c56a489c23204604e7e559ba1cfc4bf5f819399e8550

File: cluster-provision/README.md [hunk 5]
```diff
@@ -16,16 +16,16 @@
...
+* `kubevirtci/k8s-1.13.3:`: `sha256:dc76c8063994010e0cd000ea37605234f9b38d2c603522c87b5b48b9af3c8181`
```
Comment (slintes) [inline cluster-provision/README.md]: sha256:df0a214a5bacfbe71122ea62f43f6f6817a6ecab0cfc0bfbebe4eb270e8990ac

File: cluster-provision/README.md [hunk 6]
```diff
@@ -16,16 +16,16 @@
...
+* `kubevirtci/k8s-multus-1.13.3:`: `sha256:aad319b97ad0a21a1f5083100672234d7b2bd240c60a0911991a71673dcc6b90`
```
Comment (slintes) [inline cluster-provision/README.md]: sha256:c44da4b3c045808057743b666fb1966d69c0098e1028dce3f3842105b89b9a97

File: cluster-provision/README.md [hunk 7]
```diff
@@ -16,16 +16,16 @@
...
+* `kubevirtci/k8s-genie-1.11.1:`: `sha256:79af5f1fed33b78a0d1e475f902925f44bb099e44811d38caeb2460ea5d82d6d`
```
Comment (slintes) [inline cluster-provision/README.md]: sha256:791ddfacc6de2aeaf3bec9fdd16efc18140e92584ee9a6360deecd103ad784f5

**Comment by qinqon** [comment]
@slintes nooooo :-) ok will do

**Comment by slintes** [comment]
finally! Thanks for your patience, and sorry for all the hassle this currently needs.

/lgtm
/approve

**Pull request merged**